src/main/scala/strawman/collection/Seq.scala

```diff
@@ -72,6 +72,14 @@ trait IndexedSeqOps[+A, +CC[X] <: IndexedSeq[X], +C] extends Any with SeqOps[A,
     def length: Int = self.length
     def apply(i: Int): A = self(i)
   }
+
+  /** A collection containing the last `n` elements of this collection. */
+  override def takeRight(n: Int): C = fromSpecificIterable(view.takeRight(n))
```

Comment:
The implementation in the sub class will use the view implementation returned by ```view``` which is tailored for indexed views (instead of the inherited implementation which will use the one for plain iterables). I simply made it analogous to the existing reverse method.